draft-ietf-quic-recovery.md

```diff
@@ -615,6 +615,8 @@ are described in this section.
 bytes_in_flight:
 : The sum of the size in bytes of all sent packets that contain at least
   one retransmittable frame, and have not been acked or declared lost.
+  The size does not include IP or UDP overhead.  Ack only frames do not
+  count towards byte_in_flight.
```

Comment:
Ouch, that's kinda ugly.  You are saying that if you include a PING, plus a 999 octet ACK, then you have 1000 octets in flight, but drop the ping and you have 0.  I wonder what impact that has on traffic analysis.

I understand that TCP doesn't do this, but is there any reason why we need to copy that?  A packet that is sent is still creating congestion.

In any case, I think that this needs to be clearer.  This counts the entire QUIC packet, including the header and any overhead added by packet protection or integrity.  QUIC packets that only contain ACK frames are not counted.

What - if any - impact does PADDING have on this?